### PR TITLE
Added Optimism Unidex Dex

### PR DESCRIPTION
**Pull request by ARDev097**
Added trades data for unidex dex.

### CODE REVIEW

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @ARDev097 

[Sample unidex_optimism_trades](https://dune.com/queries/3309032)

File: models/unidex/optimism/unidex_optimism_trades.sql [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    unique_key=['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index'],
+    post_hook='{{ expose_spells(\'["optimism"]\',
+                                "project",
+                                "unidex",
+                                \'["ARDev097"]\') }}'
+    )
+}}
+
+{% set project_start_date = '2023-03-08' %}
+{% set firebird_finance_optimism_evt_trade_tables = [
```
Comment (jeff-dude) [inline models/unidex/optimism/unidex_optimism_trades.sql:17]:
```suggestion
{% set unidex_optimism_evt_trade_tables = [
```

File: models/unidex/optimism/unidex_optimism_trades.sql [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    source('unidex_optimism', 'metaaggregator_settlement_evt_Trade')
+] %}
+
+with dexs as (
+    {% for evt_trade_table in firebird_finance_optimism_evt_trade_tables %}
```
Comment (jeff-dude) [inline models/unidex/optimism/unidex_optimism_trades.sql:22]:
```suggestion
    {% for evt_trade_table in unidex_optimism_evt_trade_tables %}
```

**Review by jeff-dude — CHANGES_REQUESTED** [review]
minor changes requested below

**Comment by ARDev097** [comment]
Hello @jeff-dude, I have made the requested changes. Please let me know if anything else is required from my end.

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**